11_child_other.do

```diff
@@ -6,12 +6,17 @@
 *c_ITN	Child slept under insecticide-treated-bednet (ITN) last night.
     gen c_ITN = .
 	
-	capture confirm variable ml0
+	capture confirm variable ml10
 	if _rc == 0 {
-	replace c_ITN=(ml0==1) 								
```

Comment:
However, you still changed ml0 to ml10 here, which deleted the cases where ml0 is also working for other surveys where ml0 is indeed used as the c_ITN information source. To ensure the template is not only working for the specific survey you are working on but also working for other surveys, please do not replace the original ml0 to ml10 here. Please let me know if you have further questions on that, we can have a brief meeting if you think it's necessary. @yining-sun

Reply:
@wengxyu1030 Thanks Aline. Updated accordingly.

Reply:
Just to put this out there, each hiccup along the way is a step towards a smooth workflow. Just in case you have hard feelings, overall, very good work, and please let us know when you have invoice and ready to share. Cheers